src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/mediation/MediationFailedCallbackHandler.java

```diff
@@ -126,6 +133,12 @@ private CallbackResponse assignCaseState(CallbackParams callbackParams) {
             .build();
     }
 
+    private void offlineDQCheck(CallbackParams callbackParams, Claim claim) {
```

Comment:
This method doesn't really do any checking,  is the name or the logic incorrect?

Reply:
I changed the logic but forgot to change the name, thanks for pointing it out!